internal/build/docker.go

```diff
@@ -189,79 +191,87 @@ func (l *localDockerBuilder) buildFromDfWithFiles(ctx context.Context, df Docker
 	return getDigestFromAux(*result)
 }
 
-func (l *localDockerBuilder) execStepsOnImage(ctx context.Context, img digest.Digest, steps []model.Cmd) (digest.Digest, error) {
-	imageWithSteps := img
-	for _, s := range steps {
-		cId, err := l.startContainer(ctx, containerConfigRunCmd(imageWithSteps, s))
-		if err != nil {
-			return "", fmt.Errorf("startContainer '%s': %v", img, err)
-		}
-
-		id, err := l.dcli.ContainerCommit(ctx, cId, types.ContainerCommitOptions{})
-		if err != nil {
-			return "", fmt.Errorf("containerCommit: %v", err)
-		}
-		imageWithSteps = digest.Digest(id.ID)
+func (l *localDockerBuilder) execInContainer(ctx context.Context, cID containerID, cmd model.Cmd) error {
+	created, err := l.dcli.ContainerExecCreate(ctx, string(cID), types.ExecConfig{
+		Cmd: cmd.Argv,
```

Comment:
tesseract code for this sets a few additional args, are those necessary? http://bit.ly/2Lbq86u

Reply:
no, we don't want to attach stdout or tty here